core/src/main/java/com/amplifyframework/Platform.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework;
+
+/**
+ * Enum to represent various platforms that use Amplify library for
+ * tracking usage metrics.
+ */
+public enum Platform {
+    /**
+     * This is the default platform that will be included in every user-agent
+     * that is generated by this library.
+     * A user should never be using this enum to specify their platform version
+     * to avoid redundancy.
+     */
+    ANDROID("amplify-android"),
+
+    /**
+     * The Flutter library calls on Android Amplify. This enum should be specified
+     * during the construction of an Amplify configuration object to indicate that
```

Comment:
If you talk about construction, include a link to the method used to do so.

Reply:
Added sample usage code